### PR TITLE
chore(windows): Update pt-pt translation files

### PR DESCRIPTION
**Pull request by FranciscoMatos1**
@keymanapp-test-bot skip

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by mcdurdin** [comment]
@FranciscoMatos1 thank you for your contribution!

@darcywong00 can you process this and figure how best to transfer these translations to CrowdIn and manage review (quite a lot of strings have changed)?

Note: GitGuardian detection is a false positive. It is _not_ a secret.

**Review by darcywong00 — APPROVED** [review]
Looks good to me. I've manually uploaded your changes to Crowdin

https://crowdin.com/editor/keyman/428/en-pt?view=side-by-side&filter=basic&value=3

We'll merge this PR once we transition to 17.0 Beta

**Pull request merged**
